vent/helpers/meta.py

```diff
@@ -181,20 +182,74 @@ def Jobs():
         pass
 
     # get finished jobs
-    try:
-        d_client = docker.from_env()
-        c = d_client.containers.list(all=True,
-                                     filters={'label': 'vent-plugin'})
-        files = []
-        for container in c:
-            jobs[3] += 1
-            if 'file' in container.attrs['Config']['Labels']:
-                if container.attrs['Config']['Labels']['file'] not in files:
-                    files.append(container.attrs['Config']['Labels']['file'])
-        jobs[2] = len(files) - jobs[0]
-        jobs[3] = jobs[3] - jobs[1]
-    except Exception as e:  # pragma: no cover
-        pass
+    #try:
+    d_client = docker.from_env()
+    c = d_client.containers.list(all=True,
+                                 filters={'label': 'vent-plugin'})
+
+    file_names = []
+    tool_names = []
+    finished_jobs = []
+    path_dirs = PathDirs()
+    manifest = os.path.join(path_dirs.meta_dir, "status.json")
+
+    if os.path.exists(manifest):
+        file_status = 'a'
```

Comment:
if the file exists, append.
if the file doesn't exist, write.
This way, new information is added to the bottom rather than rewriting the entire file.